e2e/testcases/acme_test.go

```diff
@@ -190,6 +192,18 @@ func TestAcmeCorpRepo(t *testing.T) {
 	nt.WaitForRepoSyncs()
 }
 
+// TestObjectInCMSNamespace will test that user can sync object to CMS namespace
+func TestObjectInCMSNamespace(t *testing.T) {
+	nt := nomostest.New(t, ntopts.SkipMonoRepo, ntopts.Unstructured)
+
+	nt.RootRepos[configsync.RootSyncName].Copy("../../examples/object-in-cms-namespace", "acme")
```

Comment:
This is actually test data, not example. I would suggest you move to the `e2e/testdata` directory.

Reply:
moved to testdata folder